modules/test/playwright/fixtures/featureFlags.fixture.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {test} from '@playwright/test';
+
+exports.test = test.extend({
+	_withFeatureFlags: async ({_apiHelpers}, use) => {
+		await use(async (featureFlags, code) => {
+			for (const featureFlag of featureFlags) {
+				await _apiHelpers.featureFlag.updateFeatureFlag(
+					featureFlag,
+					'true'
+				);
+			}
+			await code();
```

Comment:
you need a try/finally here to make sure you disable the FF no matter what, right?

Reply:
See https://github.com/liferay-objects/liferay-portal/pull/2966/commits/778f2a752aad2e2477a1f1f61509fa41cdaeb9aa.